src/common/config/include/config.h.in

```diff
@@ -1,12 +1,38 @@
 #pragma once
 
-namespace config {
+namespace config
+{
+    constexpr auto PROJECT_NAME = "@PROJECT_NAME@";
+    constexpr auto VERSION = "@VERSION@";
 
-constexpr auto PROJECT_NAME = "@PROJECT_NAME@";
-constexpr auto VERSION = "@VERSION@";
+    namespace agent
+    {
+    constexpr auto DEFAULT_SERVER_URL = "@DEFAULT_SERVER_URL@";
+    constexpr auto DEFAULT_REGISTRATION_URL = "@DEFAULT_REGISTRATION_URL@";
+    constexpr auto DEFAULT_RETRY_INTERVAL = @DEFAULT_RETRY_INTERVAL@;
+    }
 
-constexpr auto BUFFER_SIZE = @BUFFER_SIZE@;
-constexpr auto DEFAULT_FILE_WAIT = @DEFAULT_FILE_WAIT@;
-constexpr auto DEFAULT_RELOAD_INTERVAL = @DEFAULT_RELOAD_INTERVAL@;
+    namespace logcollector
+    {
+        constexpr auto BUFFER_SIZE = @BUFFER_SIZE@;
```

Comment:
Note: In the future we need to evaluate if this setting should also be included in the `wazuh-agent.yml` file.